### ISSUE TITLE
关于修改主干网络后的预训练模型问题

### ISSUE THREAD

**Issue by phb-shiyige-fw** [PaddlePaddle/PaddleOCR#9808]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment： Win10
- 版本号/Version：Paddle：2.3.2  
- PaddleOCR：2.6 
- 问题相关组件/Related components： det_r50_vd_db.yml
- 运行指令/Command Code： train.py
- 完整报错/Complete Error Message：

我想在r50vd的基础上进行主干网络的修改，那么就意味着官方给的预训练模型将不会有较好的引导训练作用，我想使用迁移学习的方法，自己训练一个基于r50vd的在icdar2015基础上的预训练模型，请问配置文件中的pretrain_models应该是什么都不写，还是放入ResNet50_vd_ssld_pretrained（从det_r50_vd_db_v2.0_train的log中看到的）呢？希望能给我一下从头训练的意见和提示，谢谢！

此处附上我修改后的配置文件：
Global:
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 20
  save_model_dir: ./output/det_r50_db_icdar2015_1/
  save_epoch_step: 1200
  # evaluation is run every 2000 iterations
  eval_batch_step: [2000,2000]
  cal_metric_during_train: False
  pretrained_model: ./pretrain_models/ResNet50_vd_ssld_v2_pretrained
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: ResNet_vd
    layers: 50
    dcn_stage: [False, True, False, True]
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.000125
  regularizer:
    name: 'L2'
    factor: 0
#  lr:
#    name: Cosine
#    learning_rate: 0.00005
#    warmup_epoch: 2
#  regularizer:
#    name: L2
#    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.7
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: C:/Users/PHB/Desktop/ICDAR/icdar2015/text_localization/
    label_file_list:
      - C:/Users/PHB/Desktop/ICDAR/icdar2015/text_localization/train_icdar2015_label.txt
    ratio_list: [0.5]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [640, 640]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 2
    num_workers: 8

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: C:/Users/PHB/Desktop/ICDAR/icdar2015/text_localization/
    label_file_list:
      - C:/Users/PHB/Desktop/ICDAR/icdar2015/text_localization/test_icdar2015_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
#          limit_side_len: 1760
#          limit_type: max
          image_shape : [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 8

**Comment by phb-shiyige-fw** [comment]
希望能得到回复，感激不尽！ @LDOUBLEV

**Comment by Abraham-2020** [comment]
不使用预训练权重就不用写 pretrained_model: None

**Comment by phb-shiyige-fw** [comment]
> 不使用预训练权重就不用写 pretrained_model: None

主要是不清楚是否可以使用啊？

**Comment by phb-shiyige-fw** [comment]
> 不使用预训练权重就不用写 pretrained_model: None

我的问题关键在是否应该使用，而不是不使用时应该如何填写，您可能误解我意思了

**Comment by Abraham-2020** [comment]
当您改变自定义模型的网络结构后，如果继续使用预训练权重，可能会遇到以下问题：

1. 预训练权重可能无法适应新的网络结构。预训练权重是基于先前的网络结构进行训练的，如果您更改了网络结构，则权重可能不太适用于您的新模型。这可能导致模型的性能下降，或者需要更长的时间来收敛。
2. 预训练权重可能不包含您添加的新层或新参数的信息。如果您添加了新层或更改了参数，这些信息可能不包含在预训练权重中。这可能导致模型无法充分利用新的信息，并可能降低模型的性能。
3. 过度拟合。如果您使用预训练权重，但没有足够的数据来支持这种权重的使用，可能会导致过度拟合。这意味着模型在训练数据上表现良好，但在未见过的数据上表现不佳。
因此，当您改变自定义模型的网络结构后，最好重新训练模型，或使用其他技术来改进模型的性能。如果您决定继续使用预训练权重，则需要小心地调整模型的参数，并进行一些调整来确保模型可以充分利用预训练权重，并且不会过度拟合。

**Comment by phb-shiyige-fw** [comment]
@LDOUBLEV 希望您能给我回复，谢谢！
